pkgs/applications/radio/sdrangel/default.nix

```diff
@@ -0,0 +1,81 @@
+{ stdenv, mkDerivation, fetchFromGitHub,
+cmake, pkgconfig, glew, opencv3, libusb, libopus,
+boost, qtbase, qtwebsockets, qtmultimedia, wrapQtAppsHook,
+fftw, fftwFloat,
+limesuite,
+ocl-icd,
+libpulseaudio, libav, libiio
+}:
```

Comment:
Could you please give every argument a dedicated line? it's easier to read this way and it's a common best-practice.

Reply:
Idk most package don't do that so i don't really get why, should i really ?
it seems like it would be harder to read, you tell me ^

Reply:
I'm just speaking out of experience as a somewhat experienced new packages contributor. Maintainers with write permissions to this repository are rather wishful as for the consistency of the expressions formatting in this repo. I'm just trying to save you some criticism which will probably come in the future so you'll be more ready when a high profile member will reach you.

Reply:
Ok thank you, i'll do that, it is my first pr so i'm here to learn ^

Reply:
It's OK, this community is very friendly and the maintainers are very gentle and it's very hard to insult / hurt people's feelings in this manner. See [this](https://discourse.nixos.org/t/why-not-use-yaml-for-configuration-and-package-declaration/1333/20) as an evidence I like to exemplify.

It's just very hard with so much work to be done for things to be consistent so I'm just trying to help. Actually, as we speak, I'm having a [discussion about consistency in regard to Python 2/3](https://github.com/NixOS/nixpkgs/pull/67233#discussion_r317353914) since I'm not sure what's the best way to go in this sense.